app/frontend/src/components/forms/submission/StatusPanel.vue

```diff
@@ -369,14 +361,21 @@ export default {
             );
           }
 
-          if (this.note) {
+          if (this.emailComment) {
+            let formattedComment;
+            if (this.statusToSet === 'ASSIGNED') {
+              formattedComment = `Email to ${this.assignee.email}: ${this.emailComment}`;
+            } else if (this.statusToSet === 'REVISING' || this.statusToSet === 'COMPLETED') {
+              formattedComment = `Email to ${this.submissionUserEmail}: ${this.emailComment}`;
+            }
+
             const submissionStatusId =
               statusResponse.data[0].submissionStatusId;
             const user = await rbacService.getCurrentUser();
             const noteBody = {
               submissionId: this.submissionId,
               submissionStatusId: submissionStatusId,
-              note: this.note,
+              emailComment: formattedComment,
```

Comment:
Should the note attribute be changing to emailComment? FormService.addNote focuses on notes, of which formattedComment just happens to be for email context; but it's still a note.